exchanges/btse/btse_websocket.go

```diff
@@ -361,23 +361,23 @@ func (b *BTSE) orderbookFilter(price, amount float64) bool {
 }
 
 // GenerateDefaultSubscriptions Adds default subscriptions to websocket to be handled by ManageSubscriptions()
-func (b *BTSE) GenerateDefaultSubscriptions() ([]subscription.Subscription, error) {
+func (b *BTSE) GenerateDefaultSubscriptions() (subscription.List, error) {
 	var channels = []string{"orderBookL2Api:%s_0", "tradeHistory:%s"}
```

Comment:
This is just a comment to put it out into the world, not something for this PR
Orderbook WS has been its own WS connection endpoint since April 2023. So `orderBookL2Api` returns nothing

Reply:
🐿️ Squirrelled. Thanks.